src/hooks/usePotentialMarkets.tsx

```diff
@@ -47,37 +46,28 @@ export const PotentialMarketsProvider = ({ ...props }) => (
 
 export const usePotentialMarkets = () => useContext(PotentialMarketsContext);
 
-
-const EXCHANGE_CONFIG_FILE_PATH = '/configs/otherMarketExchangeConfig.json';
-const POTENTIAL_MARKETS_FILE_PATH = '/configs/otherMarketParameters.json';
+const POTENTIAL_MARKETS_FILE_PATH = '/configs/otherMarketData.json';
 
 export const usePotentialMarketsContext = () => {
   const stringGetter = useStringGetter();
-  const [potentialMarkets, setPotentialMarkets] = useState<PotentialMarketItem[]>();
-  const [exchangeConfigs, setExchangeConfigs] = useState<Record<string, ExchangeConfigItem[]>>();
+  const [potentialMarkets, setPotentialMarkets] =
+    useState<(NewMarketProposal & { baseAsset: string })[]>();
```

Comment:
nit: Seeing `NewMarketProposal & { baseAsset: string }` being used a lot, would it make sense to make a new type for it? e.g `NewMarketProposalWithBaseAsset`